rootfs/etc/nginx/lua/configuration.lua

```diff
@@ -177,6 +185,14 @@ local function handle_backends()
     return
   end
 
+  local timestamp = os.time()
+  success, err = configuration_data:set("timestamp", timestamp)
+  if not success then
+    ngx.log(ngx.ERR, "dynamic-configuration: error setting sync timestamp: " .. tostring(err))
```

Comment:
Please include some information in the error message about the potential consequence of not being able to update this timestamp.